README.md

```diff
@@ -23,6 +23,41 @@ scoop bucket add platformsh https://github.com/platformsh/homebrew-tap.git
 scoop install platform
 ```
 
+### Bash installer
+
+Use the bash installer for an automated installation, using the most preferable way for your system.
+
+```console
+curl -fSsl https://raw.githubusercontent.com/platformsh/cli/main/installer.sh | bash
```

Comment:
The `-l` option is an FTP thing, I guess you meant `-L` (following redirects)

Reply:
Yes, re-used this in several places now by including the `GITHUB_TOKEN` as well.

https://github.com/platformsh/cli/blob/68ece4d3b440e9e26a6da00b1de5d3b8b72efd2a/installer.sh#L149-L152